monitoring/definitions/postgres.go

```diff
@@ -0,0 +1,98 @@
+package definitions
+
+import (
+	"time"
+
+	. "github.com/sourcegraph/sourcegraph/monitoring/monitoring"
```

Comment:
```suggestion
	"github.com/sourcegraph/sourcegraph/monitoring/monitoring"
```

it's explicitly `monitoring` everywhere else, we should keep it that way for consistency, but also I feel it's better for discoverability (leads more directly to the new package documentation)

Reply:
Yeah, seeing how this is already the style. I would argue this might be a case when it is appropriate though since > 50% of lines need the import.